src/lib.rs

```diff
@@ -1,38 +1,43 @@
 #![allow(non_snake_case)] // for our name MFEKmath
-
-pub mod vector;
-pub mod piecewise;
-pub mod rect;
-pub mod bezier;
 pub mod arclenparameterization;
+pub mod bezier;
 pub mod consts;
+pub mod coordinate;
+#[cfg(feature = "default")]
+pub mod dash_along_path;
 pub mod evaluate;
 pub(crate) mod fixup;
+pub mod get_control_points;
+pub mod glyphbuilder;
+pub mod interpolator;
 pub mod parameterization;
+#[cfg(feature = "default")]
+pub mod pattern_along_path;
+pub mod piecewise;
 pub mod polar;
 pub mod primitive;
-pub mod coordinate;
-pub mod interpolator;
-pub mod glyphbuilder;
-#[cfg(feature="default")]
+pub mod rect;
+pub mod create_bazier;
```

Comment:
This is misspelled, but the module name is pretty confusing, I suggest you call it `fit_to_points` or something, and move the module `get_control_points` to be under `fit_to_points`.

Reply:
Changed it  👍.